SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -857,6 +865,44 @@ FString FSpatialGDKEditorToolbarModule::GetOptionalExposedRuntimeIP() const
 	}
 }
 
+void FSpatialGDKEditorToolbarModule::UpdateMobileClient() const
+{
+	const USpatialGDKEditorSettings* SpatialGDKSettings = GetDefault<USpatialGDKEditorSettings>();
+	const UIOSRuntimeSettings* IOSRuntimeSettings = GetDefault<UIOSRuntimeSettings>();
+	const FString ProjectName = FApp::GetProjectName();
+
+	const FString CommandLineArgs = FString::Printf(TEXT( "../../../%s/%s.uproject %s -game -log -workerType UnrealClient"), *ProjectName, *ProjectName, *(SpatialGDKSettings->ExposedRuntimeIP));
+	const FString Filename = FPaths::Combine(*FPaths::ProjectLogDir(), TEXT("ue4commandline.txt"));
+	if(!FFileHelper::SaveStringToFile(CommandLineArgs, *Filename, FFileHelper::EEncodingOptions::ForceUTF8WithoutBOM))
+	{
+		UE_LOG(LogSpatialGDKEditorToolbar, Error, TEXT("Failed to write command line args to file: %s"), *Filename);
+		return;
+	}
+
+	FString DeploymentServerExe = FPaths::ConvertRelativePathToFull(FPaths::Combine(FPaths::EngineDir(), TEXT("Binaries/DotNET/IOS/deploymentserver.exe")));
+	FString DeploymentServerArguments = FString::Printf(TEXT("copyfile -bundle \"%s\" -file \"%s\" -file \"/Documents/ue4commandline.txt\""), *(IOSRuntimeSettings->BundleIdentifier), *Filename);
+	FString DeploymentServerOutput;
+	int32 ExitCode;
+
+#if PLATFORM_WINDOWS
+	FSpatialGDKServicesModule::ExecuteAndReadOutput(DeploymentServerExe, DeploymentServerArguments, FPaths::EngineDir(), DeploymentServerOutput, ExitCode);
+#elif PLATFORM_MAC
+	FString MonoExe = FPaths::ConvertRelativePathToFull(FPaths::Combine(FPaths::EngineDir(), TEXT("Binaries/ThirdParty/Mono/Mac/bin/mono")));
+	DeploymentServerArguments = FString::Printf(TEXT("%s %s"), *DeploymentServerExe, *DeploymentServerArguments);
+	FSpatialGDKServicesModule::ExecuteAndReadOutput(MonoExe, DeploymentServerArguments, FPaths::EngineDir(), DeploymentServerOutput, ExitCode);
+#endif
+
+	if (ExitCode != 0)
+	{
+		UE_LOG(LogSpatialGDKEditorToolbar, Warning, TEXT("Failed to run %s with error %s."), *DeploymentServerExe, *DeploymentServerOutput);
+		return;
+	}
+
+	IPlatformFile& PlatformFile = FPlatformFileManager::Get().GetPlatformFile();
+	PlatformFile.DeleteFile(*Filename);
```

Comment:
This has a bool return no? Check against it because who knows what might happen.